deployment/constructor_params/lynx/infraction.yml

```diff
@@ -0,0 +1,18 @@
+deployment:
+  name: infraction
+  chain_id: 80002
+
+artifacts:
+  dir: ./deployment/artifacts/
+  filename: infraction.json
+
+constants:
+  # See deployment/artifacts/lynx.json
+  COORDINATOR_PROXY: "0xE9e94499bB0f67b9DBD75506ec1735486DE57770"
+  TACO_CHILD_PROXY: "0x42F30AEc1A36995eEFaf9536Eb62BD751F982D32"
+
+contracts:
+  - InfractionCollector:
+      proxy:
+        constructor:
+          _data: $encode:initialize,$COORDINATOR_PROXY,$TACO_CHILD_PROX
```

Comment:
```suggestion
          _data: $encode:initialize,$COORDINATOR_PROXY,$TACO_CHILD_PROXY
```

Reply:
^ In case testing was an issue you can test this using a local deployment (this should have failed locally). Happy to do a joint session with you about testing it.

Reply:
Yep. Just ping me.